theme.css

```diff
@@ -24,6 +24,51 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 ────────────────────────────────────────────────────── */
 
+/* @settings
+
+name: Royal Velvet
+id: royal-velvet
+settings:
+  -
+    id: inline-document-title-color
+    title: Inline Document Title Color
+    description: Change the color of the inline document title to match another heading or a custom hue.
+    type: class-select
+    allowEmpty: false
+    default: doc-title-h1
+    options:
+      -
+        label: Disabled
+        value: doc-title-disable
+      -
+        label: Accent Color (hue)
+        value: doc-title-accent
+      -
+        label: match headings
+        value: doc-title-match
+      -
```

Comment:
This option is not self-explanatory. Also I don't see an improvement over using the hue option.

Reply:
I'm open to alternative names, I couldn't come up with anything better than this.
The difference between this and the hue option is that this will alter the base colors (pink, orange, yellow, ...) application-wide, whereas the hue option will not touch them and instead will only alter the gradient of the document title, using the colors a +/- 30 degrees from the accent color's hue.

I can add clearer explanation to readme if you think it's needed.

Reply:
I say lets cut this option. It does not provide meaningful changes and could lead to overchoice.

Reply:
sure, I'll revert this one.

Reply:
reverted in https://github.com/caro401/royal-velvet/pull/28/commits/56d04f008b1768bfef8466252cd2f39564824880